### PR TITLE
Dw 3004 dks integration

### PR DESCRIPTION
**Pull request by UdayChokkam**
Initial pull request to integrate with DKS

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -42,3 +42,6 @@ dks.log.*
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+# Kotlin
+.kotlintest/
```
Comment (dan-hill2802) [inline .gitignore:47]: needs new line

File: src/main/kotlin/uk/gov/dwp/dataworks/encryptionmaterialsprovider/DWEncryptionMaterialsProvider.kt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    override fun getEncryptionMaterials(): EncryptionMaterials {
+        throw UnsupportedOperationException("RSA Key pair is not initialised.")
+    }
+
+    override fun getEncryptionMaterials(materialsDescription: MutableMap<String, String>?): EncryptionMaterials {
+        check(materialsDescription != null) { "Cannot handle materialsDescription as it is null" }
+        return materialsResolver.getEncryptionMaterials(materialsDescription as MutableMap<String, String?>)
+    }
+}
```
Comment (dan-hill2802) [inline src/main/kotlin/uk/gov/dwp/dataworks/encryptionmaterialsprovider/DWEncryptionMaterialsProvider.kt:41]: needs new line

**Review by dan-hill2802 — COMMENTED** [review]
Why have you deleted the README.md ?

**Comment by benj-fletch** [comment]
Closing as superceded by https://github.com/dwp/emr-encryption-materials-provider/pull/6

**Pull request closed without merging**